.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: python
 python:
   - "3.6"
+before_install:
+  - "sudo apt-get install python-enchant"
```

Comment:
is this still required even if we don't run spellcheck on travis?

Reply:
Yes, because we have `sphinxcontrib-spelling` in `requirements-optional.txt` and that requires this lib. I agree it is not optimal. Another way I see here is to alter Makefile so it asks for that that dependency if you attmpt to run the spell check.

Reply:
Ok, not a blocker, let's decide later.